cibuildwheel/linux.py

```diff
@@ -24,6 +24,8 @@ def get_python_configurations(build_selector):
         PythonConfiguration(identifier='cp36-manylinux_i686', path='/opt/python/cp36-cp36m'),
         PythonConfiguration(identifier='cp37-manylinux_i686', path='/opt/python/cp37-cp37m'),
         PythonConfiguration(identifier='cp38-manylinux_i686', path='/opt/python/cp38-cp38'),
+        PythonConfiguration(identifier='pp272-manylinux_pypy_x86_64', path='/opt/python/pp272-pypy_41'),
+        PythonConfiguration(identifier='pp372-manylinux_pypy_x86_64', path='/opt/python/pp372-pypy3_72'),
```

Comment:
@mattip If PyPy releases a Python 3.6 and 3.7 version in the future, these identifiers won't be enough to choose which wheels to build?

Reply:
Where is the ABI tag in the identifier? I see it in the path. Adding it would clarify the situation. While the python version is controlled by the python tools pip or setuptools, the ABI tag is completely under the implementation's control, so a combination of the two would map cleanly to a specific wheel

Reply:
I think @joerick's original idea was that he wanted to build all the ABI versions for a certain version (`cp27` selects both the `cp27m` as well as `cp27mu` version to be built), but maybe that's a bit more tricky here?
Or maybe we should do the same, and build multiple PyPy ABI versions for each Python version?

I'll pass this on to @joerick, as he might have a better idea of where he initially wanted to go with these build identifiers.

Reply:
Does Pypy keep ABI stability between python versions e.g. something tagged with `pp372`? Or is ABI compatibility contingent on the `pypy3_72` bit?

Reply:
The short answer to the question is that pypy keeps abi stability as long as the abi tag is the same, which is contingent on the pypy3_72 bit.

The longer answer is that as laid out in [PEP 425](https://www.python.org/dev/peps/pep-0425/#use) a spec can be broken into 3 parts: "python tag", "abi tag", and "platform tag". Since this part is platform-specific, only the python tag and abi tag are useful here. In this case the python tag is pp372, the abi tag is pypy3_72. Soon that will change, the python tag will become {pp3 or pp36 or pp37}, and the ABI tag will become pypy36_pp73 for python3.6 PyPy version 73. The idea is that the python tag reflects the language standard implemented, and the ABI tag maps to version-specific c-extension modules (dll, so, or dynlib). PyPy would like to stabilize its ABI compatibility, but it is much harder than reaching language parity, so the ABI tag changes more often than we would like.

Reply:
Ah, I see. Hm. Build identifiers are a little different to ABI tags, since we try to avoid them changing when a version is updated, since people will have baked them into config files. Right now I _think_ we're unlikely to ship a version of cibuildwheel with supporting multiple pypy versions for the same language spec, so I think these should be `pp27-` and `pp36-`. Unless you disagree about multiple pypy version support @YannickJadoul ?

Reply:
(sorry for resurrecting an old thread!)

Reply:
Well, PyPy seems to release ABI-breaking versions quite a bit more often than CPython, so I don't know how happy users would be with just building wheels for the latest PyPy ABI tag?

So that was my reasoning; but I see your point. Maybe we can have multiple lines with the same `pp27` build identifier (just like we also build two wheels for `cp27` on manylinux, `cp27m` and `cp27mu`)?

Reply:
Though, ofc, 7.3.0 and 7.2.0 PyPy versions are more than just ABI tags, I'd argue.

But maybe we should get the opinion of the PyPy developers on this?

@mattip, sorry for bother you with this again, but do you have a preference on which wheel versions you would like people to upload to PyPI, on the long run? Just the latest one? Or will this cause problems by forcing users to often update to a newer version?